oshi-json/src/main/java/oshi/json/hardware/HWDiskStore.java

```diff
@@ -73,6 +73,8 @@
      *            Number of writes to the disk
      * @param writeBytes
      *            Number of bytes written to the disk
+     * @param queueLength
+     *            Number of I/O outstanding operations (waiting + running)
```

Comment:
More a question than a comment. Have you confirmed this interpretation (wait + run) is valid on all OS's?

Reply:
Well, for Windows and FreeBSD the definition does not define anything. But I found some hints on the net that it's only the Waiting I/Os ... but this Linux statement makes me wonder about how it's defined on Linux:

> Incremented as requests are given to appropriate struct request_queue and decremented as they finish.

When do they finish? If the are completed to / from disk or if they are dispatched to disk ...

Reply:
I think you've got it mostly correct, then...